### PR TITLE
add components and design

### PR DESCRIPTION
**Pull request by raghadtt**

### CODE REVIEW

File: src/components/Message.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const Message = (props: MessageProps) => {
+  return (
+    <div>
+      <Space size={"middle"} style={{ marginBottom: 10 }}>
+        <Avatar
+          icon={<UserOutlined />}
+          style={{
+            backgroundColor:
+              props.message.type === "receiver" ? "#2A9978" : "default",
```
Comment (IbrahimLama) [inline src/components/Message.tsx]: use the enum MessageType.Receiver

Comment (IbrahimLama) [inline src/components/Message.tsx]: try to use styled-components to handle the color based on props

File: src/components/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import styles from "@/styles/Chat.module.css";
+import { AudioFilled, SendOutlined } from "@ant-design/icons";
+import { useState } from "react";
+
+interface InputProps {
+  sendMsg: (msg: string) => void;
+}
+
+const InputMsg = (props: InputProps) => {
+  const [audioStatus, setStatus] = useState(false);
```
Comment (IbrahimLama) [inline src/components/Input.tsx]: can you change audioStatus to something more clear, something similar to isRecording

File: src/pages/api/fetchResponse.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { BE_BASE_URL } from "@/utils/constants";
+import axios from "axios";
+
+export const fetchResponse = (url: string, msg: string, callback: any) => {
+  const mainUrl = BE_BASE_URL + url;
+  axios
+    .post(mainUrl, { prompt: msg })
+    .then((response) => {
+      console.log(response.data.message);
+      callback(response.data.message);
```
Comment (IbrahimLama) [inline src/pages/api/fetchResponse.tsx]: return the response.data and use async await instead of the callback function

Comment (raghadtt) [reply src/pages/api/fetchResponse.tsx]: async await function return promise

**Pull request merged**